doc/tutorial/fcm_specifics

```diff
@@ -0,0 +1,1484 @@
+                                            And committed the change.
+                                        </li>
+                                        <li>We then merged "branch1" into "branch2".
+                                            This created a Text conflict.
+                                        </li>
+                                    </font>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <code>fcm st</code> returns the following when there is a text conflict on the
+                repository:
+                <pre>C    statusesC_text.txt </pre>
+                How did we resolve the text conflict?
+                <div class="panel-group" id="accordion">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h4 class="panel-title">
+                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseEight">
+                                <span class="glyphicon"></span>
+                                Answers
+                                </a>
+                            </h4>
+                        </div>
+                        <div id="collapseEight" class="panel-collapse collapse">
+                            <div class="panel-body">
+                                <div class="row">
+                                    <font color="green">
+                                    <code>fcm conflicts</code>
+                                    Left clicked on the text option that we want to keep.
+                                    <br/>
+                                    Then press "m", or use the toolbar; File -> Exit with MERGED M to merge the 
+                                    change.
+                                    </font>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                Now we need to learn more about the command <code>fcm conflicts</code>.
+            </div>
+            <div class="well">
+                <h4>conflicts</h4>
+                The command <code>fcm conflicts|cf</code> is a graphical tool to help
+                resolve conflicts within the local/working copy.
+                Note it cannot help with all conflicts.
+                It works with the most effectively with Test Conflicts.
+                <br/>
+                <br/>
+                A type of conflict that we may come across are "Text conflicts". 
+                These are returned via <code>fcm st</code> as
+                <pre>C    statusesC_text.txt </pre>
+                where column 1 has a "C" in it.
+                <br/>
+                To resolve the text conflict we can use the 3-way difference tool xxdiff.
```

Comment:
... we can use a 3-way difference tool such as xxdiff.

Following sentence suggests xxdiff is optional.